lms/djangoapps/teams/models.py

```diff
@@ -183,16 +191,25 @@ def reset_team_size(self):
         self.save()
 
 
+@python_2_unicode_compatible
 class CourseTeamMembership(models.Model):
     """
     This model represents the membership of a single user in a single team.
 
     .. no_pii:
     """
 
-    def __unicode__(self):
+    def __str__(self):
         return "[CourseTeamMembership user={}, team={}]".format(self.user, self.team)
 
+    def __repr__(self):
```

Comment:
What is the reason to change the serialization of these objects from having angle brackets to parentheses?